tests/apollo/clientservice/config/clientservice.config

```diff
@@ -29,6 +29,7 @@ tls_1_3_cipher_suite_list: TLS_AES_256_GCM_SHA384
 tls_1_3_comm_enabled: true
 tls_certificates_folder_path: /concord/tls_certs
 tls_cipher_suite_list: TLS_AES_256_GCM_SHA384
+use_unified_certificates: true
```

Comment:
You cannot enable use_unified_certificates without enabling enable_multiplex_channel, too.